crates/api/src/community/transfer.rs

```diff
@@ -40,7 +41,11 @@ impl Perform for TransferCommunity {
       CommunityModeratorView::for_community(context.pool(), community_id).await?;
 
     // Make sure transferrer is either the top community mod, or an admin
-    if local_user_view.person.id != community_mods[0].moderator.id
+    if local_user_view.person.id
+      != community_mods
+        .get(0)
+        .map(|cm| cm.moderator.id)
+        .unwrap_or(PersonId(0))
```

Comment:
This makes the code unreadable. Store it in a variable first or make a helper method.